corehq/apps/geospatial/forms.py

```diff
@@ -73,12 +81,24 @@ class Meta:
         #       '<a href="{}" target="_blank">support documentation</a>.'),
         #     'https://confluence.dimagi.com/pages/viewpage.action?pageId=164694245'
         # ),
-        choices=GeoConfig.VALID_DISBURSEMENT_ALGORITHMS,
+        choices=DISBURSEMENT_ALGORITHM_OPTIONS,
         required=True,
     )
+    mapbox_token = forms.CharField(
+        label=_("Enter mapbox token"),
+        help_text=_(
+            "Enter your Mapbox API token here. Make sure your token has the correct scope configured"
+            " for use of the Mapbox Matrix API."
+        ),
+        required=False,
+    )
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        if toggles.SUPPORT_ROAD_NETWORK_DISBURSEMENT_ALGORITHM.enabled(self.domain):
```

Comment:
Were we able to confirm if we need to add a new feature flag for this?

Reply:
We don't need it, it just seemed too tedious to remove all references from history for what it's worth, so I took the lazy route and left it as is, since I am removing it [here](https://github.com/dimagi/commcare-hq/pull/34114/commits/8b72bf665ea192904652c52bf986d49a9aa06c7a).

Let me know what you think...if you think "no, let's remove it from history" I'm happy to, otherwise I'll just leave it.

Reply:
No worries. I assumed that with the commit clean up, this would not get re-introduced.

Reply:
Just commenting here that [a new FF is indeed now needed](https://dimagi.atlassian.net/browse/SC-3486).

Reply:
that was quite a ride. This feature flag made it way into the code.